### PR TITLE
Validators ui

### PR DESCRIPTION
**Pull request by PanDeBatalla94**

### CODE REVIEW

File: src/main/java/com/fundation/search/view/MainView.java [hunk 1]
```diff
@@ -9,11 +9,7 @@
 import java.awt.Dimension;
 import java.awt.Container;
 import java.awt.BorderLayout;
-import javax.swing.JFrame;
-import javax.swing.JTabbedPane;
-import javax.swing.JMenu;
-import javax.swing.JMenuBar;
-import javax.swing.JPanel;
+import javax.swing.*;
```
Comment (manu863018) [inline src/main/java/com/fundation/search/view/MainView.java]: Import should not be to everything *

File: src/main/java/com/fundation/search/view/PanelDate.java [hunk 2]
```diff
@@ -11,9 +11,7 @@
 import info.clearthought.layout.TableLayout;
 import info.clearthought.layout.TableLayoutConstraints;
 
-import javax.swing.JPanel;
-import javax.swing.JCheckBox;
-import javax.swing.JLabel;
+import javax.swing.*;
```
Comment (manu863018) [inline src/main/java/com/fundation/search/view/PanelDate.java]: Import should not be to everything *

File: src/main/java/com/fundation/search/view/PanelMultimediaSearch.java [hunk 3]
```diff
@@ -3,7 +3,14 @@
 import info.clearthought.layout.TableLayout;
 import info.clearthought.layout.TableLayoutConstraints;
 
-import javax.swing.*;
```
Comment (manu863018) [inline src/main/java/com/fundation/search/view/PanelMultimediaSearch.java]: Import should not be to everything *

File: src/main/java/com/fundation/search/view/PanelNamePath.java [hunk 4]
```diff
@@ -9,12 +9,8 @@
 import info.clearthought.layout.TableLayout;
 import info.clearthought.layout.TableLayoutConstraints;
 
-import javax.swing.JLabel;
-import javax.swing.JTextField;
-import javax.swing.JPanel;
-import javax.swing.JFileChooser;
-import javax.swing.JButton;
-import javax.swing.JCheckBox;
+import javax.swing.*;
```
Comment (manu863018) [inline src/main/java/com/fundation/search/view/PanelNamePath.java]: Import should not be to everything *

File: src/main/java/com/fundation/search/view/PanelMultimediaSearch.java (hunk 3 above)
Comment (PanDeBatalla94) [reply src/main/java/com/fundation/search/view/PanelMultimediaSearch.java]: this was removed

**Pull request merged**